bin/auto-pr.ps1

```diff
@@ -88,13 +88,21 @@ function execute($cmd) {
     return $output
 }
 
+function _selectMasterBranch {
+    $branches = execute 'hub branch --all'
+    $master = if ($branches -like '*/origin/main') { 'main' } else { 'master' }
+
+    return $master
+}
+
```

Comment:
I think this should be more generic and not only `main` or `master`

```suggestion
function _selectMasterBranch {
    $branches = execute 'hub branch --all'
    $branch = $branches | Select-String -Pattern 'origin/HEAD -> origin/(.+?)$'
    if ($branch) {
        return $branch.Matches.Groups[1].Value
    } else {
        Write-Error 'HEAD branch not found'
        exit 1
    }
}
```

Reply:
There is no reason to support something else.

Reply:
well, if someone's default branch is not `main` or `master` the script execution fails... it's not likely, but you never know.

If someone wants it's default branch to be called `default` instead of `main` (or master)...

Anyway, it's just a suggestion to make the script more generic.